README.md

```diff
@@ -67,6 +67,30 @@ export class Renderer extends Component {
 <this.myComponent />
 ```
 
+### Using existing components
+
+`ember-repl` is strict-mode only, so any component that you want to invoke 
+needs to be passed to the scope option of `compileHBS` or `compileJS`.
+Following code is assuming that right next to our `Renderer` component
+there is a component named `Bar`.
+
+```js
+import Component from '@glimmer/component';
+import { compileHBS } from 'ember-repl';
+import BarComponent from './bar'
```

Comment:
This imports the whole component, right? Not only template as I suggested elsewhere.

Reply:
Exactly., in Octane+, all components are class+template, set build transform or manually via [set component template](https://emberjs.github.io/rfcs/0481-component-templates-co-location.html#low-level-primitives)